### DISCUSSION TITLE
tvm安装报错

### DISCUSSION THREAD

**Discussion by WuZheng326** [mlc-ai/mlc-zh#122] [Q&A]
$make
[  0%] Built target tvm_libinfo_objs
[ 90%] Built target tvm_objs
[ 91%] Performing configure step for 'project_libbacktrace'
/bin/sh: /home/wuzheng/Documents/DNN_AutoSchedule/software/tvm/cmake/libs/../../3rdparty/libbacktrace/configure: No such file or directory
make[2]: *** [CMakeFiles/project_libbacktrace.dir/build.make:127: libbacktrace/src/project_libbacktrace-stamp/project_libbacktrace-configure] Error 127
make[1]: *** [CMakeFiles/Makefile2:432: CMakeFiles/project_libbacktrace.dir/all] Error 2
make: *** [Makefile:160: all] Error 2

tvm的子模块都下载了，不知道为什么一直报错说没有文件，环境是centos，想请教下这个问题应该怎么解决呢？

**Comment by yzh119** [comment]
能否确认一下`3rdparty/libbacktrace`这个目录下是否有文件呢

**Comment by WuZheng326** [comment]
你好，我是吴峥，已收到您的邮件，我会尽快给您回复。

**Comment by 0xAyin** [comment]
建议虚拟机重新搭建一个centos环境，复现一下问题
